src/payment/braintree.md

```diff
@@ -5,7 +5,7 @@ title: Braintree
 {:.bs-callout-warning}
 **Deprecation Notice** <br/>
 Due to the Payment Service Directive [PSD2]({% link stores/compliance-payment-services-directive.md %}) and the continued evolution of many APIs, this payment integration is at risk of becoming outdated and no longer security compliant. For this reason, it is now deprecated and we are recommending that you disable it in your Magento 2.3.x configuration and transition to the official Braintree payment extension from [Magento Marketplace](https://marketplace.magento.com/catalogsearch/result/?q=braintree). <br/><br/>
-**This integration has been deprecated from current versions of 2.3.x.**<br/><br/>
+**This integration has been deprecated from current versions of 2.4.x.**<br/><br/>
```

Comment:
Since this is for the 2.3 guide and we have a replacement topic for the VBE in the 2.4 docs, I propose the following:

```suggestion
**This integration has been deprecated from current versions of 2.3.x. and replaced with a different Braintree integration in 2.4.0 and later.**<br/><br/>
```